mcs/class/Mono.Compiler/Mono.Compiler.BigStep/BigStep.cs

```diff
@@ -99,7 +99,7 @@ class Builder {
 				}
 				IntPtr fnptr = LLVM.GetPointerToGlobal (engine, Function);
 				unsafe {
-					result = new NativeCodeHandle ((byte*)fnptr, -1, methodInfo);
+					result = new NativeCodeHandle ((byte*)fnptr, -1);
```

Comment:
Can also drop the `methodInfo` arg from Finish.   anything we need about the source method should've gone into the builder state before the call to `Finish` already.